### ISSUE TITLE
2019parr rankings inconsistent with bracket

### ISSUE THREAD

**Issue by MC42** [the-blue-alliance/the-blue-alliance-data#67]
Could the rankings calculation task be triggered please?

**Comment by ZachOrr** [comment]
Specifically, what's wrong with the rankings?

**Comment by MC42** [comment]
834 is listed as rank 2.  At the event, this would make them either the first pick of the first alliance, or the second rank alliance captain as this event used a standard playoff bracket.  (8 teams, 3 alliances, etc.)  

Alliance 1 is shown as being 2539, with 834 not on their alliance at all.   The #2 seed alliance is 1218 with 834 also not present.  The playoff bracket, which as far as I know is derived from the playoffs order.

Rewatching the alliance selection video, this is confirmed. (2539 ranked #1, 2607 first overall pick from #2)

**Comment by fangeugene** [comment]
Is the bracket wrong or are the rankings wrong?
If it's the rankings, then we can't do anything because we don't calculate
the rankings, they are submitted to us.
If it's the bracket, then we can't do anything either because the event
must have played the matches in the wrong order compared to what's standard.

**Comment by ZachOrr** [comment]
Yeah - this looks like they just played playoff matches out-of-order, and there’s no proper alliance data for the event. @fangeugene Will the playoff bracket look at the alliances to generate, or just the matches/order? Wondering if we backfilled the alliance information if we could fix this

**Comment by fangeugene** [comment]
We only look at the matches / order. Maybe we could fill in the alliance information and then file an issue to use it to generate the bracket?

@MC42 can you give me a list of the alliances?

**Comment by MC42** [comment]
After watching the playoffs, this is the bracket as I saw it/

Alliance | Captain | First Pick | Second Pick
-- | -- | -- | --
1 | 2539 | 2607 | 484
2 | 1218 | 1640 | 293
3 | 5407 | 316 | 2199
4 | 834 | 1676 | 2191
5 | 5401 | 7414 | 272
6 | 1391 | 1712 | 9539 (2539B)
7 | 423 | 1111 | 6326
8 | 433 | 5420 | 321

Is it possible some qualification matches are missing, the team list is definitely inconsistent given that there are 23 valid teams in playoffs and 19 in the list....

**Comment by fangeugene** [comment]
Wait, so the bracket is correct. It's the rankings that are wrong.

**Comment by MC42** [comment]
More or less, a remap for one team might be handy but otherwise it's correct.

**Comment by fangeugene** [comment]
Yeah then we really can't do anything about it unless someone has the correct rankings.

**Comment by MC42** [comment]
I'll ask around, is our ranking calculator broken for some reason for 2019?

**Comment by fangeugene** [comment]
We've never had a ranking calculator.

**Comment by MC42** [comment]
Sorry, poor choice of words.  I meant the sort order criteria for offseasons in 2019, I had thought we mirrored the FIRST rankings for official events, but that seemed to have worked correctly for other offseasons.  I'll ask the volunteer how the rankings data was uploaded to TBA.

**Comment by fangeugene** [comment]
I don't get what you mean. Ranking data is 100% provided by the offseason (whether running FMS or Cheesy Arena or something else). TBA doesn't derive ranking data from match results.

**Comment by MC42** [comment]
My sincere apologies, I had a very wrong memory of what the rankings helper handles.  Going to see about talking to the event people and seeing if there's a rankings file anywhere to try and get it corrected.  Thanks for the help
